### DISCUSSION TITLE
This plugin doesn't work with rpoxy

### DISCUSSION THREAD

**Discussion by RomanAleynikov** [YiiGuxing/TranslationPlugin#5192] [Q&A] [answered]
Hello everybody! I use proxy in IDE and this plugin doesn't work for this reason. I can add plugin domain in proxy exception but I don't know what is domain. Can anybody help me?

**Comment by YiiGuxing** [comment]
Which translation engine are you using?

**Reply by RomanAleynikov** [reply]
Microsoft translator and Google translator.

**Reply by YiiGuxing** [reply, marked as answer]
- Google  translate:
  - translate.googleapis.com
- Microsoft translator:
  - api.cognitive.microsofttranslator.com
  - edge.microsoft.com

**Reply by RomanAleynikov** [reply]
It has worked! Very thank you!
